shoppingcart/Dockerfile

```diff
@@ -1,8 +1,10 @@
 FROM python:3.8.0-slim
+
```

Comment:
I tried to build this docker image and ran it, but I got:
<img width="696" alt="Screenshot 2020-05-20 at 01 09 25" src="https://user-images.githubusercontent.com/12616445/82388179-3585dc80-9a39-11ea-88a1-d9aa952c5494.png">

Tried to find out why, but did not get to it.